Dockerfile

```diff
@@ -5,8 +5,11 @@ LABEL maintainer="PnT DevOps Automation - Red Hat, Inc." \
       summary="Image used to run Angular tests." \
       distribution-scope="public"
 
+COPY ca.crt /opt/ca.crt
+
 # Configure the chrome executable for Karma
 ENV CHROME_BIN=chromium-browser
+ENV NODE_EXTRA_CA_CERTS=/opt/ca.crt
```

Comment:
Please combine this with the previous line to reduce the number of layers.

Reply:
Fixed.